Resources/doc/usage.rst

```diff
@@ -1,7 +1,7 @@
 Usage
 =====
 
-Add these two lines in your layout:
+In applications not using WebPack add these two lines in your layout:
```

Comment:
I think we write "Webpack" (see: https://en.wikipedia.org/wiki/Webpack).

Reply:
Ok for all lowercase then

Reply:
This is still not conform to the discussed capitalisation

Reply:
Sorry @tobias-93 fixed now!